sdk/ai/azure-ai-assistants/samples/async_samples/sample_assistant_basics_async.py

```diff
@@ -71,19 +71,9 @@ async def sample_assistant_basic_operation():
         message = await assistant_client.create_message(thread_id=thread.id, role="user", content="Hello, tell me a joke")
         logging.info(f"Created message, message ID: {message.id}")
 
-        run = await assistant_client.create_run(thread_id=thread.id, assistant_id=assistant.id)
+        run = await assistant_client.create_run_and_poll(thread_id=thread.id, assistant_id=assistant.id, interval=4)
```

Comment:
We currently have create_and_process_run method which handles the poll + tool calls (if toolset has been used in assistant creation) and let's not yet add this poll method.
We might want to think more about existing create_and_process_run method support.

Reply:
Done